src/middleware/index.js

```diff
@@ -10,7 +10,19 @@ const handle500 = ({ status = 500, name, message }, _req, res, next) => {
     res.status(status).json({ name, statusCode: status, message })
 }
 
+const checkEmailPasswordExist = (req, res, next) => {
```

Comment:
I'm not sure this needs to be extracted into middleware. When will we ever need to check the existence of an email & password combo again? `Register a user` maybe... but, there, we would also need to check stuff like firstName, lastName and track. A more reusable approach would be to use something like [express-validator](https://express-validator.github.io/docs/).